provider/ecs/ecs.go

```diff
@@ -189,10 +191,20 @@ func (p *Provider) loadECSConfig(ctx context.Context, client *awsClient) (*types
 
 	instances = fun.Filter(p.filterInstance, instances).([]ecsInstance)
 
+	services := make(map[string][]ecsInstance)
+
+	for _, i := range instances {
```

Comment:
could you rename `i` to `instance` ?

I know it's already use but I think `i` is an ambiguous name in a `for`.